EZSwiftExtensionsTests/ArrayTests.swift

```diff
@@ -285,6 +285,24 @@ class ArrayTests: XCTestCase {
         XCTAssertEqual(duplicatedElementsArrayOrderMaintained.unique(), [2, 1, 3])
     }
     
+    func testStep() {
+        XCTAssertNil(self.numberArray.step(-1))
+        XCTAssertNil(self.numberArray.step(0))
+
+        XCTAssertEqual(self.numberArray.step(2)![0], [0, 1])
+        XCTAssertEqual(self.numberArray.step(2)![1], [2, 3])
+        XCTAssertEqual(self.numberArray.step(2)![2], [4, 5])
+        XCTAssertEqual(self.numberArray.step(2)![3], [1])
```

Comment:
In the `setup()` function, the numberArray == `[0, 1, 2, 3, 4, 5, 1]`

There are 7 digits so the last array will have only 1 entry for steps of 2 or 3.

I think `[1]` will be better than `[1, nil]` or `[1, nil, nil]`